jangjuseong/controllers/postController.js

```diff
@@ -0,0 +1,48 @@
+const postService = require('../services/postService');
+
+const createPost = async (req, res) => {
+  const { title, content, postImgUrl, userId } = req.body;
```

Comment:
클라이언트가 보내는 요청 중 데이터베이스에서 반드시 필요로 하는 데이터가 존재할 것입니다. 
반드시 필요한 데이터가 우리 서버에 제대로 요청되지 않았을 때의 예외처리를 해주세요!

```suggestion
  const { title, content, postImgUrl, userId } = req.body;
  
  if (!title | content | postImgUrl | userId) {
    throw new Error("Key error!");
  }
```